CMakeLists.txt

```diff
@@ -73,3 +73,8 @@ install(DIRECTORY   ${PROJECT_SOURCE_DIR}/source/external/imgui
         DESTINATION include
         PATTERN     "src" EXCLUDE
 )
+
+# Install Vulkan Validation Layer Settings
+install(FILES   ${PROJECT_SOURCE_DIR}/vk_layer_settings.txt
```

Comment:
Since its installing to the sample dir, I think this should be in the sample's cmake.